src/frontend/screens/Library/index.tsx

```diff
@@ -34,9 +34,253 @@ import { hasHelp } from 'frontend/hooks/hasHelp'
 import EmptyLibraryMessage from './components/EmptyLibrary'
 import CategoriesManager from './components/CategoriesManager'
 import LibraryTour from './components/LibraryTour'
+import AlphabetFilter from './components/AlphabetFilter/AlphabetFilter'
 
 const storage = window.localStorage
 
+const filterByPlatformHelper = (
```

Comment:
this refactor in this file look really overkill to reuse some function, this is not right